File: src/socket/src/models/ManualScopeRunGraphApprovalRequest.ts
import BaseGraphApprovalBotRequest from "@/models/bases/BaseGraphApprovalBotRequest";
import { Entity } from "typeorm";

@Entity({ name: "manual_scope_run_graph_approval_request" })
class ManualScopeRunGraphApprovalRequest extends BaseGraphApprovalBotRequest {}

export default ManualScopeRunGraphApprovalRequest;

File: src/shared/py/langboard_shared/domain/models/ManualScopeRunGraphApprovalRequest.py
from .bases import BaseGraphApprovalBotRequest
from .GraphApprovalRequest import GraphApprovalOriginType


class ManualScopeRunGraphApprovalRequest(BaseGraphApprovalBotRequest, table=True):
    @staticmethod
    def get_request_type() -> GraphApprovalOriginType:
        return GraphApprovalOriginType.ManualScopeRun
